### PR TITLE
Reduce bundle size

### PR DESCRIPTION
**Pull request by savelichalex**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by vetkamak — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -47,23 +47,22 @@
...
         "currency-formatter": "1.5.4",
-        "dayjs": "1.9.1",
         "fontfaceobserver-es": "3.3.3",
         "fuse.js": "3.4.6",
         "html2canvas": "1.0.0-rc.7",
-        "libphonenumber-js": "1.7.14",
-        "lodash": "4.17.21",
         "mobile-detect": "1.4.3",
         "nanoid": "3.1.23",
-        "prop-types": "15.6.2",
```
Comment (sertony) [inline package.json:66]: 👍

File: packages/components/src/UICompatibilityView/index.web.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// @flow
+import React from 'react';
+import { Platform, StyleSheet, Text, View } from 'react-native';
+// TODO: we have `mobile-detect` in UIDevice
+// maybe use only one lib for detecting?
+import Bowser from 'bowser';
```
Comment (sertony) [inline packages/components/src/UICompatibilityView/index.web.js]: AFAIR we used this to identify the IE.

File: packages/navigation_legacy/package.json [hunk 3]
```diff
@@ -37,12 +37,10 @@
         "@tonlabs/uikit.components": "^2.3.0",
         "@tonlabs/uikit.core": "^2.3.0",
         "@tonlabs/uikit.hydrogen": "^2.3.0",
-        "prop-types": "15.6.2",
         "react-image-lightbox": "5.1.1",
```
Comment (sertony) [inline packages/navigation_legacy/package.json]: Need not to forget removing it ;)

File: packages/navigation_legacy/src/UIImageView/index.js [hunk 4]
```diff
@@ -23,11 +23,18 @@ let Lightbox;
 let LightboxMobile;
 
 if (Platform.OS === 'web') {
-    require('./LightboxStyle');
-    Lightbox = require('react-image-lightbox').default;
+    // TODO: we should use our new Lightbox built with reanimated here
+    // but since UIImageView isn't used anywere for now
```
Comment (sertony) [inline packages/navigation_legacy/src/UIImageView/index.js]: How about killing UIImageView completelly with the lightbox dependencies? =)

File: packages/navigation_legacy/package.json [hunk 5]
```diff
@@ -37,12 +37,10 @@
...
         "react-native-country-picker-modal": "git+https://github.com/tonlabs/react-native-country-picker-modal.git",
         "react-native-lightbox": "0.7.0",
```
Comment (sertony) [inline packages/navigation_legacy/package.json]: Same

**Review by SAnatoliiS — APPROVED** [review]

File: packages/navigation_legacy/package.json (hunk 3 above)
Comment (savelichalex) [reply packages/navigation_legacy/package.json]: What do you mean? :)

File: packages/keyboard/src/useCustomKeyboard.tsx [hunk 6]
```diff
@@ -80,10 +76,10 @@ export function useCustomKeyboard(
                 setCustomKeyboard(undefined);
             }, duration);
         };
-        Keyboard.addListener('keyboardWillHide', callback);
+        const keyboardSubscription = Keyboard.addListener('keyboardWillHide', callback);
 
         return () => {
-            Keyboard.removeListener('keyboardWillHide', callback);
+            keyboardSubscription.remove();
```
Comment (sertony) [inline packages/keyboard/src/useCustomKeyboard.tsx:82]: Thanks!

File: packages/navigation_legacy/package.json (hunk 3 above)
Comment (sertony) [reply packages/navigation_legacy/package.json]: Just kill it from the project =)

Comment (savelichalex) [reply packages/navigation_legacy/package.json]: Oh, I thought you commented on `prop-types` :D

File: packages/keyboard/src/useCustomKeyboard.tsx (hunk 6 above)
Comment (savelichalex) [reply packages/keyboard/src/useCustomKeyboard.tsx:82]: Anyway it didn't help and I still see that annoying warning 🤬

**Review by sertony — APPROVED** [review]

**Pull request merged**